app/screens/ContactPickerScreen.tsx

```diff
@@ -51,9 +51,8 @@ export const ContactPickerScreen: FC<ContactPickerScreenProps> = observer(
     const [custom, setCustom] = useState([])
 
     const navigation = useNavigation<any>()
-    const contacts = useUserContacts()
 
-    const data = contacts.concat(custom)
+    const data = custom.concat(getContacts)
```

Comment:
if you need to do this then there is a bug saving contacts that has to be fixed in arclib.

Reply:
ah, this screen is just invite screen, when user add custom contact, it just add it to selected list to send invite, it not add follow to this custom contact

I put custom.contact before it will show custom on top of the list instead of bottom